lib/libspl/Makefile.am

```diff
@@ -42,6 +42,6 @@ nodist_libspl_la_SOURCES = \
 	$(USER_ASM) \
 	$(KERNEL_C)
 
-libspl_la_LIBADD = -lrt
+libspl_la_LIBADD = -lrt $(TIRPC_LIBS)
```

Comment:
@ryao my first inclination was that we shouldn't do that since it will change the public libspl interface.  However, to my knowledge there aren't any consumers other than `nvpair.c` and this would simplify the code and minimize dependencies which is a good thing.  I'd be OK with moving the `xdr_control()` function in to `nvpair.c` and declaring it static.

This would somewhat break the symetry between `spl.ko` and `libspl.so` but in this context I think it makes sense.